arnold/crud/read/case.py

```diff
@@ -0,0 +1,13 @@
+"""CRUD module to find entries."""
+
+
+from arnold.adapter import ArnoldAdapter
+from arnold.models.database.case import Case
+
+
+def get_case(case_id: str, adapter: ArnoldAdapter) -> Case | None:
+    """Retrieve a case document from the database."""
```

Comment:
Maybe mention that it raises a Pydantic error if the case can not be parsed correctly.